code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -266,6 +266,11 @@
 	volume = 10
 	proj_piercing = 1
 
+/obj/item/reagent_containers/syringe/piercing/ninja
```

Comment:
I would recommend maybe giving ninjas a way to synthethize these so they aren't reliant on scavenging.

Reply:
maybe fore 200E or something cuz its really powerfull